lib/jit_preloader/active_record/base.rb

```diff
@@ -48,6 +51,15 @@ def reload(*args)
       end
 
     end
+
+  end
+
+  def self.generate_key(method_name, conditions)
```

Comment:
@doliveirakn - this is the part I mentioned that worried me a little. I wanted to abstract out the key generation, but I'm wary of adding a method like this to ActiveRecord::Base. We could namespace it like `jit_generate_key` or something I guess

As I said I don't mess around with object internals much in Ruby so there might be an even better way of doing this.